Test/Utils/ComputeNode/Service.ps1

```diff
@@ -48,6 +46,14 @@ NSSM output: $($Output.Output)
 "@
         throw [HardError]::new($ExceptionMessage)
     }
+
+    ForEach ($Pair in $AdditionalParams.GetEnumerator()) {
```

Comment:
```suggestion
    ForEach ($Key in $AdditionalParams.Keys) {
        $Pair = $AdditionalParams[$Key]
```

Don't know why, but GetEnumerator doesn't work sometimes as intented - it returns type instead of value.